tests/test_20_use_case_topology.py

```diff
@@ -515,16 +515,114 @@ def test_070_port_uni_up_no_l2vpn_associated(self):
         final_data = requests.get(API_URL).json()
         assert final_data[l2vpn_id] == l2vpn_data['data'], "L2VPN state changed unexpectedly"
 
-    def test_100_vlan_range_change(self):
+    @pytest.mark.xfail(reason="The L2VPN with VLAN 1060 remains up even after modifying the VLAN range to [100–200]")
+    def test_106_removing_vlan_uni(self):
         """
         Use Case 10: OXPO sends a topology update with a changed VLAN range is for any of the services supported.
         """
-        
-    def test_110_service_no_longer_supported(self):
+        interfaces_id = 'aa:00:00:00:00:00:00:01:50'
+        interface_name = 'Ampath1-eth50'
+        ampath_api = KYTOS_API % 'ampath'
+        api_url_ampath = f'{ampath_api}/topology/v3'
+        payload = {
+            "sdx_vlan_range": [[100,200], [1000, 2000]]
+        }
+        response = requests.post(f"{api_url_ampath}/interfaces/{interfaces_id}/metadata", json=payload)
+        assert response.status_code == 201, response.text
+
+        # Force to send the topology to the SDX-LC
+        sdx_api = KYTOS_SDX_API % 'ampath'
+        response = requests.post(f"{sdx_api}/topology/2.0.0")
+        assert response.status_code == 200, response.text
+
+        time.sleep(15)
+
+        response = requests.get(API_URL_TOPO)
+        data = response.json()
+        for node in data['nodes']: 
+            for port in node['ports']:
+                if port['name'] == interface_name:
+                    assert port['services']['l2vpn_ptp']['vlan_range'] == [[100,200], [1000, 2000]], port
+
+        l2vpn_data = self.create_new_l2vpn(vlan='1060')
+        l2vpn_id = l2vpn_data['id']
+
+        # Removing vlan
+        payload = {
+            "sdx_vlan_range": [[1100, 1500]]
+        }
+        response = requests.post(f"{api_url_ampath}/interfaces/{interfaces_id}/metadata", json=payload)
+        assert response.status_code == 201, response.text
+
+        # Force to send the topology to the SDX-LC
+        sdx_api = KYTOS_SDX_API % 'ampath'
+        response = requests.post(f"{sdx_api}/topology/2.0.0")
+        assert response.status_code == 200, response.text
+
+        time.sleep(15)
+
+        response = requests.get(API_URL)
+        assert response.status_code == 200, response.text
+        l2vpn_response = response.json()
+        l2vpn_response = l2vpn_response.get(l2vpn_id)
+        assert l2vpn_response.get("status") == "error", l2vpn_response
+
+    @pytest.mark.xfail(reason="The L2VPN path associated with VLAN 1070 remains unchanged after modifying the VLAN range to [100–200]")
+    def test_107_removing_vlan_nni_with_alternate_path(self):
```

Comment:
Gretel, it is not clear to me the difference from this test 107 and the test `test_102_shrinking_vlan_nni_with_alternate_path`